mesheryctl/internal/cli/root/system/start.go

```diff
@@ -60,7 +61,7 @@ var startCmd = &cobra.Command{
 		}
 		hc, err := NewHealthChecker(hcOptions)
 		if err != nil {
-			return errors.Wrapf(err, "failed to initialize healthchecker")
+			return handlers.ErrHealthCheckFailed(err)
```

Comment:
@metonymic-smokey Errors shud be local to every package. Can you initiate a `error.go` for this package and added new ones for these?

Reply:
@kumarabd done!
Could you check my latest commits and let me know if I got you right?